content/en/hosting/4.x/data-migration-3x-docker-to-4x-k3s-single.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Migration from 3.x Docker Compose to 4.x K3s (Single Node)"
+linkTitle: "Migration: 3.x Docker Compose to 4.x K3s (Single-Node)"
+weight: 1
+description: >
+  Guide on how to migrate existing data from CHT 3.x Docker Compose deployment to CHT 4.x single-node K3s deployment
+relatedContent: >
+---
+{{ < read-content file="hosting/4.x/_partial_migration_3x_docker_to_4x_k3s.md"  }}
+
+## Create values.yaml for K3s Deployment
+{{ < read-content file="hosting/4.x/_partial_values_explanation.md"  }}
+
+```yaml
+project_name: "your-namespace"
+namespace: "your-namespace"
+chtversion: <version>
+
+upstream_servers:
+  docker_registry: "public.ecr.aws/medic"
+  builds_url: "https://staging.dev.medicmobile.org/_couch/builds_4"
+upgrade_service:
+  tag: 0.32
+
+couchdb:
+  password: "<password>"
+  secret: "<secret>"
+  user: "<admin-user>"
+  uuid: "<uuid>"
+  clusteredCouch_enabled: false
+  couchdb_node_storage_size: 100Gi
+
+ingress:
+  host: "<url>"
+
+environment: "remote"
+cluster_type: "k3s-k3d"
+cert_source: "specify-file-path"
+certificate_crt_file_path: "<path-to-tls>/fullchain.crt"
+certificate_key_file_path: "<path-to-tls>/privkey.key"
+
+nodes:
+  node-1: "couch01"
+
+couchdb_data:
+  preExistingDataAvailable: "true"
+  dataPathOnDiskForCouchDB: "data"
+  partition: "0"
+
+local_storage:
+  preExistingDiskPath-1: "/srv/couchdb1"
+```
+
+## Deploy to K3s
+```shell
```

Comment:
Please add a comment like: 
```
Use cht-deploy (link) to deploy CHT-Core to K8s. 
```

Reply:
I've added this in the partial to avoid duplication. Right before the prepare node section.

>We are also going to utilize the `cht-deploy` script from the [cht-core](https://github.com/medic/cht-core) repo. If you don't already have that, clone it.

Still think it should come here?

Reply:
Good point, I've added a note right above it.